tests/e2e/go.mod

```diff
@@ -17,6 +17,6 @@ replace (
 
 require (
 	github.com/oasisprotocol/oasis-core/go v0.2100.1
-	github.com/oasisprotocol/oasis-sdk/client-sdk/go v0.0.0-20210319174144-fdda3859da1f
+	github.com/oasisprotocol/oasis-sdk/client-sdk/go v0.0.0-20210328195842-4de788c1c6f7
```

Comment:
how do we manage this?

Reply:
There's a replace directive in the block above, so it always uses the current files:
```
	github.com/oasisprotocol/oasis-sdk/client-sdk/go => ../../client-sdk/go
```
I just updated this because it seems nice to have a more recent hash there, even if it's unused :)

Reply:
oh ok